src/extension/features/budget/custom-average-budgeting/index.js

```diff
@@ -0,0 +1,111 @@
+import { controllerLookup, getEmberView } from 'toolkit/extension/utils/ember';
+import {
+  getEntityManager,
+  isCurrentMonthSelected,
+  isCurrentRouteBudgetPage,
+} from 'toolkit/extension/utils/ynab';
+import { formatCurrency } from 'toolkit/extension/utils/currency';
+import { Feature } from '../../feature';
+
+export class CustomAverageBudgeting extends Feature {
+  constructor() {
+    super();
+  }
+
+  shouldInvoke() {
+    return isCurrentRouteBudgetPage() && isCurrentMonthSelected();
+  }
+
+  _calculateLookback() {
```

Comment:
YNAB is using moment internally, I think instead of doing this, we should just make a new YNAB date and use it in a call to `transaction.date.isBetweenMonths`